install_test/CMakeLists.txt

```diff
@@ -1,8 +1,10 @@
-
 cmake_minimum_required(VERSION 3.8)
 
 project(InstallTest LANGUAGES CXX)
 
+# The `enable_language` is needed when the library is build static with CUDA
+#enable_language(CUDA)
```

Comment:
commented code

Reply:
Is the current version acceptable for you:
```cmake
# `enable_language(CUDA)` is needed when ginkgo was statically build with CUDA
```
I want to add a hint that the `enable_language(CUDA)` is needed if it was built as a static library.

Reply:
If Ginkgo was built as a static library? I will add some flag later as part of the CMake improvements which after doing `find_package(Ginkgo...)` will provide some information on whether it was built statically or not. Wee can improve this part by using an `if`.

Reply:
@tcojean Good to know, thanks, I added a TODO in the description, so I don't forget either.